packages/core/contracts/cross-chain-oracle/test/OracleMessengerMock.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../OracleHub.sol";
+import "../OracleSpoke.sol";
+
+/**
+ * @notice Can be used as either a Parent or Child messenger mock in unit tests for the Oracle Hub and Spoke
+ * cross-chain contracts. The reason that this can't also be used for the Governor Hub and Spoke is that the
+ * sendMessageToChild is called with different encoded data in the Oracle versus the Governor.
+ */
+contract OracleMessengerMock {
```

Comment:
we dont have an `OracleMessenger`. do you want to re-factor this?

Reply:
This contract and `GovernorMessengerMock` are intended to be used as the Messenger contracts for the `Governor` and `Oracle` hub/spoke unit tests respectively. Do you think there's a better name for this? The long name for this contract would be `MessengerMockForOracle`